Source/Atomic/Resource/Image.cpp

```diff
@@ -2034,7 +2124,12 @@ unsigned char* Image::GetImageData(Deserializer& source, int& width, int& height
 
     SharedArrayPtr<unsigned char> buffer(new unsigned char[dataSize]);
     source.Read(buffer.Get(), dataSize);
-    return stbi_load_from_memory(buffer.Get(), dataSize, &width, &height, (int*)&components, 0);
+    // #623 BEGIN TODO: Crunch expects all 4 components. This isn't ideal, but since
+    // precompressed versions should always be loaded anyway it doesn't matter.
+    // Might need to handle differently per platform.
+    components = 4;
+    return stbi_load_from_memory(buffer.Get(), dataSize, &width, &height, nullptr, components);
+    // #623 END TODO: Crunch expects all 4 components. This isn't ideal, but since

```

Comment:
This will probably be tripped by  the changes:

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/f4c3ac360ce7376a091dc714ceff33cd17415285/Source/Atomic/Resource/Image.cpp#L883

GetImageDataForceComponents with a modification to BeginLoad sounds pretty hacky, there's a better way?